pkg/ansible/watches/watches.go

```diff
@@ -321,13 +307,25 @@ func Load(path string, maxWorkers, ansibleVerbosity int) ([]Watch, error) {
 		return nil, err
 	}
 
-	watches := []Watch{}
-	err = yaml.Unmarshal(b, &watches)
+	// First we unmarshall to the alias structure
+	alias := []alias{}
+	err = yaml.Unmarshal(b, &alias)
 	if err != nil {
 		log.Error(err, "Failed to unmarshal config")
 		return nil, err
 	}
 
+	// We copy contents from alias structure to the watch structure
+	watches := []Watch{}
+	for _, a := range alias {
+		w, err := buildWatch(a)
```

Comment:
We have the new for the Watch which will set the Default values see; https://github.com/operator-framework/operator-sdk/blob/16e89924addf13eb4fcf12e3084ccec3e20db0cc/pkg/ansible/watches/watches.go#L281

So IMO we can : 
- Gen the GKV
- Create a new watch 
- Then use the alias structure to set the values

Reply:
I tried this and it was giving some errors. So I worked around that.

Reply:
I hope this implementation is fine